second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md

```diff
@@ -164,63 +176,57 @@ beta-x86_64-pc-windows-msvc
-
-After some time, once Rust developers who use nightly releases have been able
-to try out the new feature, team members will discuss the feature, how it’s
-worked out on nightly, and decide if it should make it into stable Rust or not.
-If the decision is to move forward, the feature gate is removed, and the
-feature is now considered stable! It rides the trains into a new stable release
-of Rust.
-
-## Summary
-
-You’re already off to a great start on your Rust journey! In this chapter,
-you’ve:
-
-* Learned what makes Rust unique
-* Installed the latest stable version of Rust
-* Written a “Hello, world!” program using both `rustc` directly and using
-  the conventions of `cargo`
-* Found out about how Rust is developed
-
-This is a great time to build a more substantial program, to get used to
-reading and writing Rust code. In the next chapter, we’ll build a guessing game
-program. If you’d rather start by learning about how common programming
-concepts work in Rust, see Chapter 3.
+Ahora, cada vez que llame `rustc` o `cargo` dentro de *~/projects/needs-nightly*,
+`rustup` se encargará de que usted esté usando la versión nocturna de Rust, en
+vez de su versión estable por defecto de Rust. ¡Esto es útil cuando quiere tener
+muchos proyectos de Rust!
+
+### El Proceso RFC y Los Equipos
+
+Así que ¿Cómo aprende usted de estas nuevas funcionalidades? El modelo dedesarrollo
+de Rust sigue un *Request For Comments (RFC)(Proceso de Pedir Por Comentarios)*.
+Si quiere una mejora en Rust, puede escribir una propuesta, llamada RFC.
+
+Cualquiera puede escribir RFCs para mejorar Rust, y las propuestas son revisadas
+y discutidas por el equipo de Rust, el cual está comprendido de sub-equipos
+de muchos temas. Hay una lista completa de los equipos [en Rust’s website]
+(https://www.rust-lang.org/en-US/team.html), la cual incluye equipos para cada
+area del proyecto: diseño del lenguaje, implementación del compilador,
+infraestructura, documentación, y más. Los equipos apropiados leen las propuestas
+y los comentarios, escriben algunos comentarios ellos mismo, y eventualmente,
+hay un consenso para aceptar o negar la funcionalidad.
+
+Si hay una funcionalidad aceptada, una cuestión se abre en el repositorio de Rust,
+y alguien la puede implementar. ¡La persona que la implementa puede fácilmente no
+ser la persona que propuso la funcionalidad en primer lugar! Cuando la implementación
+está lista, termina en la rama `master` detrás de una entrada de funcionalidades,
+como lo discutimos en la sección de "Funcionalidades Inestables".
+
+## Resumen
+
+¡Ya está listo para un buen comienzo en su viaje con Rust! En este capítulo usted ha:
+
+* Aprendido qué hace a Rust único
+* Instalado las últimas versiones estables de Rust
+* Escrito un programa "Hello, world!" usando `rustc` directamente y usando las
+convenciones de `cargo`
+* Encontrado sobre el cómo Rust está desarrollado
+
+Este es un buen momento para escribir un programa substancioso, para acostumbrarse
```

Comment:
'substancioso' -> 'relevante' or 'importante'

Reply:
On it.